### PR TITLE
renamed package to org.ossgang

### PR DESCRIPTION
**Pull request by andreacalia**
Co-authored-by: michi42 <michi42@users.noreply.github.com>
Co-authored-by: kaifox <kaifox@users.noreply.github.com>

### CODE REVIEW

**Comment by kaifox** [comment]
Of course ok for me ;_)

**Comment by michi42** [comment]
Same here ;-)

File: src/main/java/org/ossgang/commons/observable/DispatchingObservableValue.java [hunk 1]
```diff
@@ -48,7 +46,7 @@ protected DispatchingObservableValue(T initial) {
     @Override
     public Subscription subscribe(Observer<? super T> listener, SubscriptionOption... options) {
         Set<SubscriptionOption> optionSet = new HashSet<>(Arrays.asList(options));
-        if (optionSet.contains(FIRST_UPDATE)) {
+        if (optionSet.contains(ObservableValueSubscriptionOption.FIRST_UPDATE)) {
```
Comment (michi42) [inline src/main/java/org/ossgang/commons/observable/DispatchingObservableValue.java]: Did you remove these static imports on purpose, or was it IntelliJ?

Comment (andreacalia) [reply src/main/java/org/ossgang/commons/observable/DispatchingObservableValue.java]: Intellij.. I just did refactor :confused:

**Pull request merged**